plugins/eg-sampler.lv2/sampler_ui.c

```diff
@@ -291,11 +300,14 @@ static void
 cleanup(LV2UI_Handle handle)
 {
 	SamplerUI* ui = (SamplerUI*)handle;
-	gtk_widget_destroy(ui->box);
```

Comment:
I'd like to note that this only seems to matter when the UI is instantiated but the widget isn't immediately shown somewhere.  i.e. UI instantiated -> `show` called later from some other UI/button in the host.